packages/zend-xml/library/Zend/Xml/Security.php

```diff
@@ -164,7 +177,7 @@ public static function scanFile($file, DOMDocument $dom = null)
      *
      * @return boolean
      */
-    public static function isPhpFpm()
+    public static function isVulnerablePhpFpm()
```

Comment:
Although I agree with @leendt that the `isVulnerablePhpFpm` name would suit it better, I second @glensc we should not introduce unnecessary changes (and definitely not breaking ones).

To avoid breaking changes, we should still have `isPhpFpm()` which would be an alias to `isVulnerablePhpFpm()`, but still `isPhpFpm()` would keep its ambiguous meaning...

I'd say better to keep it as it was. I can see in the method's docblock that the purpose of it is explained, and I guess from it that the original zf1 maintainers first introduced the method as a workaround, and then they didn't care to rename it when php-fpm had been eventually fixed.

Now I wouldn't like to start any active development on this repo (like renaming stuff, adding new features, tidying up the code to make it up to current standards etc.) since the purpose of it is to just help people "stuck" with zf1 (where it's simply economically impossible to update their projects to different, more modern frameworks) to keep their systems running also on current php versions, **while also** fixing evident issues and bugs whenever possible, but always with BC intact. After all, it's simply not encouraged to start a new project based on ancient zf1, when there are so many, much better, modern alternatives...

Reply:
I agree and will revert that commit.

Active development is not my goal either.

[some bacground]
We're stuck with a behemoth of an (internal) application deeply coupled with (now stock) ZF1 1.12. It'll have to be refactored, a full rewrite or keeping up two applications is a path I want to avoid if possible due to cost and the risk of repeating old 'mistakes'.

I was very happy to find this project. Not only are most PHP 7 deprecateds/warnings and some bugs fixed, the components are also split up as far as possible. After registering ZendX with the Zend autoloader (and removing the require_onces) it was a drop in replacement and I was very surprised it also ran without major issues in PHP 8.

Using this could enable us to to take more time to 'unwire' our own code/logic from Zend1 and replace components gradually while making use of everything PHP 7 and soon 8 will offer.
[/some background]